okonf/facts/systemd.py

```diff
@@ -59,3 +59,21 @@ async def enforce(self, host: Executor) -> bool:
         async with host.lock(f"systemctl-service-{self.name}"):
             await host.check_output(f"sudo systemctl restart {self.name}")
         return True
+
+
+class TimerRegistered(Fact):
+    """Ensure that a Systemd timer is registered"""
+
+    is_stateless = True
+    name: str
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    async def enquire(self, host: Executor) -> bool:
+        return False
+
+    async def enforce(self, host: Executor) -> bool:
+        async with host.lock(f"systemctl-register-timer-{self.name}"):
```

Comment:
The name of the lock must be consistent with the other facts targeting the same service.
```suggestion
        async with host.lock(f"systemctl-service-{self.name}"):
```